static/javascript/tba_js/tba.js

```diff
@@ -153,3 +153,13 @@ $(document).ready(function(){
     $("#robot-image").html('<div class="thumbnail carousel team-media-carousel"><a href="' + url + '" target="_blank"><img src="' + url + '" alt="April Fools!"></a></div>');
   }
 });
+
+// Init Firebase
+var config = {
```

Comment:
These aren't private keys (so we're not screwed that I've already committed and pushed this), but I guess it would be nice to not hardcode these into the repo if people want to fork it. But then by that argument any reference to "tbatv-prod-hrd" and "thebluealliance.com" should be replaced during deployment 😛.

I'll see what I can do.